pycc/ccresponse.py

```diff
@@ -391,14 +504,310 @@ def r_X2(self, pertbar, omega):
 
         return r_X2
 
+    def in_Y1(self, pertbar, X1, X2):
+        contract = self.contract
+        o = self.ccwfn.o
+        v = self.ccwfn.v
+        l1 = self.cclambda.l1
+        l2 = self.cclambda.l2
+        cclambda = self.cclambda
+        t2 = self.ccwfn.t2
+        hbar = self.hbar
+        L = self.H.L
+
+        # Inhomogenous terms appearing in Y1 equations
```

Comment:
I would move this up and elaborate so that it's clear to others right away what's going on in these functions.

Reply:
I have moved it up and elaborated on the solve left function.